lock/src/main/java/com/auth0/android/lock/views/PanelHolder.java

```diff
@@ -44,10 +48,19 @@ public PanelHolder(Context context) {
         configuration = null;
     }
 
-    public PanelHolder(Context context, Bus lockBus, Configuration configuration) {
+    public PanelHolder(PasswordlessLockActivity context, Bus lockBus, Configuration configuration) {
         super(context);
         this.bus = lockBus;
         this.configuration = configuration;
+        this.usingPasswordless = true;
```

Comment:
What if the dashboard has all the 4 kinds of connections? (social/enterprise/db/passwordless), then the form will display all of those, and if instantiated from a `LockActivity` instead of a `PasswordlessLockActivity`, the passwordless flow wont work because the event subscriber method is on the `PasswordlessLockActivity`.